### ISSUE TITLE
Add image folder

### ISSUE THREAD

**Issue by sangjunp4rk** [mkabobel/colmaracademy#4]
Don’t forget to upload your image folder. Your website didn’t load correctly because all of your images and videos are missing on Github.

**Comment by mkabobel** [comment]
Yeah, I realized that today when I was rechecking my project. Sorry about that. I've added it now.
